plugin/storage/es/spanstore/schema.go

```
@@ -0,0 +1,183 @@
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+package spanstore
+
+const spanMapping = `{
+   "settings":{
+      "index.mapping.nested_fields.limit":50,
+      "index.requests.cache.enable":true,
+      "index.mapper.dynamic":false,
+      "analysis":{
+         "analyzer":{
+            "traceId_analyzer":{
+               "type":"custom",
+               "tokenizer":"keyword",
+               "filter":"traceId_filter"
+            }
+         },
+         "filter":{
+            "traceId_filter":{
+               "type":"pattern_capture",
+               "patterns":[
+                  "([0-9a-f]{1,16})$"
+               ],
+               "preserve_original":true
+            }
+         }
+      }
+   },
+   "mappings":{
+      "_default_":{
+         "_all":{
+            "enabled":false
+         }
+      },
+      "span":{
+         "properties":{
+            "traceID":{
+               "type":"string",
+               "analyzer":"traceId_analyzer",
+               "fielddata":"true"
+            },
+            "parentSpanID":{
+               "type":"keyword",
+               "ignore_above":256
+            },
+            "spanID":{
+               "type":"keyword",
+               "ignore_above":256
+            },
+            "operationName":{
+               "type":"keyword",
+               "ignore_above":256
+            },
+            "startTime":{
+               "type":"long"
+            },
+            "duration":{
+               "type":"long"
+            },
+            "flags":{
+               "type":"integer"
+            },
+            "logs":{
+               "properties":{
+                  "timestamp":{
+                     "type":"long"
+                  },
+                  "tags":{
+                     "type":"nested",
+                     "dynamic":false,
```

Maybe better use "strict" so it returns errors if there's some bug in the code and it attempts to index fields that have no mappings? false silently ignores such cases.